### ISSUE TITLE
Gleam version in default Github's test workflow does not match the gleam's version used to generate it

### ISSUE THREAD

**Issue by ducdetronquito** [gleam-lang/gleam#3385]
## TL;DR

When creating a new Gleam project with Gleam 1.3.1, the generated default Github's test workflow uses Gleam 1.3.0-rc1.

## Steps and code to reproduce the problem

Generate a new gleam project with `gleam new myproject`
The file `myproject/.github/workflows/test.yml` will contains the following content where can see `gleam-version: "1.3.0-rc1"` is used:

```yml
name: test

on:
  push:
    branches:
      - master
      - main
  pull_request:

jobs:
  test:
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v4
      - uses: erlef/setup-beam@v1
        with:
          otp-version: "26.0.2"
          gleam-version: "1.3.0-rc1"
          rebar3-version: "3"
          # elixir-version: "1.15.4"
      - run: gleam deps download
      - run: gleam test
      - run: gleam format --check src test
```

It seems that the version is defined in https://github.com/gleam-lang/gleam/blob/v1.3.1/compiler-core/src/version.rs via the `CARGO_PKG_VERSION` env variable made available by cargo during the build step.
 
## Versions for Gleam, OS, the runtime and any other software used

- Gleam 1.3.1 installed with `mise use -g gleam@1.31`
- OS: Ubuntu 22.04 on WSL 2 (Windows 11)

**Comment by lpil** [comment]
Could you run `gleam --version` and `gleam new` and include the output please. Thank you

**Comment by ducdetronquito** [comment]
My bad, I had a gleam 1.3.0-rc1 installed without mise on my computer.
Sorry for the inconvenience... :(

**Comment by lpil** [comment]
No problem! Easily done
